modules/core/src/main/java/org/apache/ignite/internal/processors/metastorage/persistence/DistributedMetaStorageHistoryItem.java

```diff
@@ -50,6 +58,8 @@ public DistributedMetaStorageHistoryItem(String key, byte[] valBytes) {
 
     /** */
     public DistributedMetaStorageHistoryItem(String[] keys, byte[][] valBytesArray) {
+        assert keys.length == valBytesArray.length;
```

Comment:
Abbreviation should be used for `valBytesArray` parameter and corresponding class field.